### PR TITLE
Update 为什么激活函数？.md

### PR DESCRIPTION
**Pull request by YapWH1208**
fixes on latex formatting problem in github

### CODE REVIEW

File: 激活函数/为什么激活函数？.md [hunk 1]
```diff
@@ -19,24 +19,28 @@ $$\large \sigma = {{1}\over{1+e^{-y}}}$$
...
 - 取值无上限
 	- 梯度爆炸现象 --> 数值初始化
-- 非零$x$值，会出现神经坏死现象
+- 非零 $x$ 值，会出现神经坏死现象
 	- 使用优化版激活函数
-		- Leaky ReLU ($0.01x$)
-		- Parametric ReLU ($\alpha x$)
+		- Leaky ReLU ( $0.01x$ )
+		- Parametric ReLU ( $\alpha x$ )
 		- 等等
```
Comment (YapWH1208) [inline 激活函数/为什么激活函数？.md]: Can be deleted

**Review by YapWH1208 — COMMENTED** [review]
delete line 等等

File: 激活函数/为什么激活函数？.md [hunk 2]
```diff
@@ -19,25 +19,28 @@ $$\large \sigma = {{1}\over{1+e^{-y}}}$$
...
 $$\large \tanh = {{1 - e^{-y}}\over{1 + e^{-y}}}$$
-- 定义域$[-1,1]$
+- 定义域 $[-1,1]$
 - 导数最大值：1
 	- 饱和函数，取值有上限
 		- 遇到极限导数链时会导致梯度消失
 
 ## ReLU
-$$\text{ReLU}(x) = \begin{cases} 0 & \text{if } x < 0 \\ x & \text{if } x \geq 0 \end{cases}$$
+$$\text{ReLU}(x) = \begin{cases}
```
Comment (YapWH1208) [inline 激活函数/为什么激活函数？.md]: make it a line

Comment (YWHerng) [reply 激活函数/为什么激活函数？.md]: multi line cases needs to be in multi line in github

**Pull request merged**